src/Sylius/Bundle/AdminBundle/templates/product/show/content/page_body/details/configurable_product/tab/product.html.twig

```diff
@@ -0,0 +1,22 @@
+{% set variant = hookable_metadata.context.variant %}
+{% set product = hookable_metadata.context.product %}
+
+<td>
+    <div class="d-flex gap-3 align-items-center">
+        <div>
+            {% if product.imagesByType('thumbnail') is not empty %}
+                {% set path = product.imagesByType('thumbnail').first.path|imagine_filter(filter|default('sylius_admin_product_thumbnail')) %}
+            {% elseif product.images.first %}
+                {% set path = product.images.first.path|imagine_filter(filter|default('sylius_admin_product_thumbnail')) %}
+            {% else %}
+                {% set path = asset('build/admin/images/200x200.png', 'admin') %}
+            {% endif %}
```

Comment:
Could be extracted into a macro, or a component for that matter.

Reply:
yeah there is existing one, as i checked now